### DISCUSSION TITLE
How to force Renovate to merge deps with major and minor updates under one group PR?

### DISCUSSION THREAD

**Discussion by simPod** [renovatebot/renovate#14822] [Request Help] [answered]
I specified this packageRule 

```json
    "packageRules": [
        {
            "matchPackagePatterns": ["\\bphpstan\\b"],
            "groupName": "phpstan packages",
            "groupSlug": "phpstan"
        }
    ],
```

to group all phpstan related deps under one PR.

It kind of works, it groups `phpstan/phpstan`, `phpstan/phpstan-mockery`, `phpstan/phpstan-phpunit`, `phpstan/phpstan-strict-rules`, `phpstan/phpstan-webmozart-assert` together.

I also see in debug logs that `thecodingmachine/phpstan-safe-rule` is part of the group.

However, I want to force all of those into the same PR but `thecodingmachine/phpstan-safe-rule` is being left out.

all `phpstan/` packages are being bumped from 0.x to 1.x (major)  
`thecodingmachine/phpstan-safe-rule` is being bumped from 1.0.x to 1.2.x

I guess that's the reason why Renovate wants to create two separate PRs (`phpstan/*` and `thecodingmachine/phpstan-safe-rule`)

Is there a way to tell Renovate to group those?

**Comment by viceice** [comment, marked as answer]
https://docs.renovatebot.com/configuration-options/#separatemajorminor

**Reply by simPod** [reply]
I'm still learning those options. Was trying `matchUpdateTypes`. Thanks!
